test/util/spec.helpers.js

```diff
@@ -0,0 +1,117 @@
+var helpers = require('../../lib/util/helpers');
+
+describe('helpers', function () {
+
+    describe('getPreviousSteps', function () {
+        var getPreviousSteps, steps;
+
+        beforeEach(function () {
+            getPreviousSteps = helpers.getPreviousSteps;
+        });
+
+        it('should expect 2 arguments', function () {
+            getPreviousSteps.length.should.be.equal(2);
```

Comment:
I agree that one should be used, not both, but would probably need to be a separate PR